### ISSUE TITLE
RequestAccessEmail not used for OnPrem-provisioning

### ISSUE THREAD

**Issue by nils-a** [SharePoint/PnP-Sites-Core#1793]
#### Category
[X] Bug
[ ] Enhancement

#### Environment
[ ] Office 365 / SharePoint Online
[ ] SharePoint 2016
[X] SharePoint 2013

If SharePoint on-premises, what's exact CU version: 

#### Expected or Desired Behavior
setting RequestAccessEmail should modify the RequestAccessEmail - even OnPrem.

#### Observed Behavior
Applying:
```xml
<pnp:Provisioning ... >
  <pnp:Templates>
    <pnp:ProvisioningTemplate ...>
      <pnp:WebSettings RequestAccessEmail="" />
```
Should disable access-requests. However, OnPrem this is not working.

**Comment by VesaJuvonen** [comment]
This is correct and by design as for the SP2013 baseline currently is April 2015 CU which does NOT support RequestAccessEmail. We are looking for updating the version, BUT - we need to do that carefully with sufficient amount of documentation and guidance or we will potentially break thousands of customers.

**Comment by nils-a** [comment]
ok. But some documentation on that topic would be nice, then... There is no hint of this fact whatsoever in the docs.
And probably some warning as a feedback that parts of the supplied template are not being processed?
